cast-dssspell.sh

```diff
@@ -7,42 +7,48 @@ CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
 MCD_ADM=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_ADM")")")
 MCD_GOV=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_GOV")")")
 MCD_IOU=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_IOU")")")
-BALANCE=$(seth call "$MCD_GOV" 'balanceOf(address)(uint256)' "$ETH_FROM")
+BALANCE_GOV=$(seth call "$MCD_GOV" 'balanceOf(address)(uint256)' "$ETH_FROM")
+BALANCE_IOU=$(seth call "$MCD_IOU" 'balanceOf(address)(uint256)' "$ETH_FROM")
 HAT=$(seth call "$MCD_ADM" 'hat()(address)')
 APPROVALS=$(seth call "$MCD_ADM" 'approvals(address)(uint256)' "$HAT")
 DEPOSITS=$(seth call "$MCD_ADM" 'deposits(address)(uint256)' "$ETH_FROM")
+HAT_THRESHOLD=$(seth --to-wei 100000 ETH)
 CONTI=1
 
 if [[ -z "$1" ]];
 then
     echo "Please specify the Goerli Spell Address"
 else
-    if [[ "$APPROVALS" -ge "$DEPOSITS" ]];
+    if [[ "$APPROVALS" -ge "$DEPOSITS" ]] || [[ "$APPROVALS" -lt "$HAT_THRESHOLD" ]];
     then
-    DELTA=$((APPROVALS - DEPOSITS))
+    GAP=$((HAT_THRESHOLD - APPROVALS))
```

Comment:
couldn't this be negative if `APPROVALS > DEPOSITS > HAT_THRESHOLD`?

Reply:
717d6bc should fix it